src/BundleAnalyzerPlugin.js

```diff
@@ -23,7 +23,18 @@ class BundleAnalyzerPlugin {
       logLevel: 'info',
       // deprecated
       startAnalyzer: true,
+
+      // JSON specific options block
+      failBuild: true,
       initialLoadingResources: [],
+      maxInitialLoadingSizeSingle: 100,
+      maxInitialLoadingSizeBundle: 500,
+      chunksLoadingResources: [],
+      maxLazyLoadingSizeSingle: 100,
+      failOnBudgetError: true,
```

Comment:
what's the diff between failBuild and failOnBudgetError?

Reply:
shouldn't we do it false by default to not break in case someone else also uses this plugin?